cpmpy/expressions/globalconstraints.py

```diff
@@ -455,7 +455,15 @@ def decompose(self):
         return cons
 
     def value(self):
-        start, dur, end, demand, cap = [argval(a) for a in self.args]
+        import numpy as np
+
+        argvals = [np.array([argval(a) for a in arg]) if is_any_list(arg)
+                   else argval(arg) for arg in self.args]
+
+        if any(a is None for a in argvals):
+            return None
+
+        start, dur, end, demand, cap = argvals
         # start and end seperated by duration
         if not (start + dur == end).all():
```

Comment:
an alternative fix could be not to assume that it is an nparray...

e.g. `if all(s + d == e for s,d,e in zip(start, dur, end))` or similar?

Reply:
Also possible, but non-numpiefied decomposition of the capacity limit is much harder imho